### ISSUE TITLE
ALT text for featured image block should be used when in query loop block

### ISSUE THREAD

**Issue by kuropixel** [Automattic/wp-calypso#73861]
### What

**ALT text** from images that are used in the "featured image" block **is not being displayed** when inside **query loop block**. Single post pages display the alt text correctly from block, but not for Index template or when the block is being used inside query loop.

### Why

From an accessibility point of view, it would be great if alt text that's added to images is also displayed in the live site's source code. For example, a site using index template as their main homepage isn't able to have alt text display on the site due to this issue which is not ideal for accessibility. 

### How

- Use a FSE theme or have featured image block inside query loop block.
- Add alt text to an image in media library
- Assign that image as featured image to post
- Alt text is not added to the image on the live site.
![1](https://user-images.githubusercontent.com/83020905/221784785-4a0c6305-0be3-401a-bf97-40afba0fa936.png)
![2](https://user-images.githubusercontent.com/83020905/221784800-051b66b4-aaed-45cb-b627-19c075b59426.png)

**Comment by cuemarie** [comment]
📌 ACTIONS
- Feature request kept

**Comment by jartes** [comment]
📌 REPRODUCTION RESULTS
- Tested on Simple – Could Not Replicate
- Tested on Atomic – Could Not Replicate
- Replicable outside of Dotcom – No

📌 ACTIONS
- Closed as could not replicate (please reopen if the issue reoccurs)
